imageroot/actions/configure-module/20config

```diff
@@ -185,6 +185,40 @@ if "timezone" in data and data["timezone"] != os.environ["WEBTOP_TIMEZONE"]:
     agent.set_env("RESTART_WEBDAV", "1")
     agent.set_env("RESTART_Z_PUSH", "1")
 
+# adjust the ejabberd module
+if "ejabberd_module" in data:
+    ejabberd_split = data["ejabberd_module"].split(',')
+    ejabberd_module = ejabberd_split[0]
+    ejabberd_domain = ejabberd_split[1]
```

Comment:
Why did you embed the module_id and the domain in the same JSON field?

Reply:
It is cleaner the use two different fields ( as defined in the JSON schema)

Reply:
yep typo, no worries I needed it for the UI